pages/docs/reason-compiler/latest/decorators.mdx

```diff
@@ -525,7 +525,14 @@ var myBook = new Book();
 
 ### `[@bs.deriving accessors]`
 
-Reason records are represented as arrays in JS.
+`bs.deriving accessors` makes it more convenient to access records.
```

Comment:
~~~suggestion
The `[@bs.deriving accessors]` can be used to either generate getter functions for a decorated **record type**, or to generate creator functions for **variant type constructors**.
~~~